src/System.Windows.Forms/src/System/Windows/Forms/Controls/ToolStrips/ToolStripHighContrastRenderer.cs

```diff
@@ -238,6 +238,7 @@ protected override void OnRenderItemText(ToolStripItemTextRenderEventArgs e)
             ((ToolStripButton)e.Item).Checked) ||
             (typeof(ToolStripMenuItem).IsAssignableFrom(e.Item.GetType()) &&
             ((ToolStripMenuItem)e.Item).DisplayStyle != ToolStripItemDisplayStyle.Image &&
+            !e.Item.IsOnDropDown &&
```

Comment:
Could we update the styling here? [Style guidelines](https://github.com/dotnet/winforms/blob/main/docs/coding-style.md#line-breaks) is `&&` and `||` be broken where they are at the beginning of the line e.g.
```suggestion
            && ((ToolStripButton)e.Item).Checked)
            || (typeof(ToolStripMenuItem).IsAssignableFrom(e.Item.GetType())
            && ((ToolStripMenuItem)e.Item).DisplayStyle != ToolStripItemDisplayStyle.Image
            && !e.Item.IsOnDropDown
```